Clarify readme for tiny example

linsun opened the pull request:
I ran into a few issues when trying the tiny example with kubernetes module only today.
1) The doc was not clear as to tiny installer vm vs creating your own installer env.
2) i assumed it was using root user to run Ansible scripts, which caused not able to SSH into the VMs issue.

MikeSpreitzer commented:
@linsun this change undoes some other recent work.  Can you start over from the current master?

MikeSpreitzer commented on examples/tiny-example.md:
```diff
@@ -97,26 +97,19 @@ most of the contents of the OpenRadiant repository in `~/openradiant`.
 
-If you are running Ubuntu on your installer, you may need to install
-the following python packages:
+If you are running Ubuntu in your host, you may need to install the following
```
This change is undoing a more recent change.

MikeSpreitzer commented on examples/tiny-example.md:
````diff
@@ -97,26 +97,19 @@ most of the contents of the OpenRadiant repository in `~/openradiant`.
-```
-
-In any case, install ansible and its `netaddr` module:
+Install ansible and its `netaddr` module:
````
This change is undoing a recent change, please don't.

MikeSpreitzer commented on examples/tiny-example.md:
```diff
@@ -136,16 +129,24 @@ pip install --upgrade ansible
 an environment.  This will create the certificates and keys that are
-common throughout the environment, and deploy the API proxy.
+common throughout the environment.  Someday soon this will also deploy
+the API proxy.   
```
This undoes a recent change.  Please don't.

MikeSpreitzer commented on examples/tiny-example.md:
```diff
@@ -136,16 +129,24 @@ pip install --upgrade ansible
+common throughout the environment.  Someday soon this will also deploy
+the API proxy.   
+
+Use vagrant ssh to ssh into the installer machine:
```
This is correct only if the installer VM was used.

MikeSpreitzer commented on examples/tiny-example.md:
```diff
@@ -136,16 +129,24 @@ pip install --upgrade ansible
 ( cd ansible; \
   ansible-playbook -v -i ../examples/envs/dev-vbox/radiant01.hosts env-basics.yml \
-      -e "envs=../examples/envs env_name=dev-vbox" )
+      -e "envs=../examples/envs env_name=dev-vbox cluster_name=dev-vbox-radiant01" )
```
The `env-basics.yml` playbook does not need the `cluster_name` variable to be set.

MikeSpreitzer commented on examples/tiny-example.md:
```diff
@@ -136,16 +129,24 @@ pip install --upgrade ansible
 
 Use Ansible on the installer machine to deploy an OpenRadiant shard on
-the target machines.
+the target machines, using the vagrant user.
```
The change adds a dangling modifier.  Did you intend "using the vagrant user" to modify "Use Ansible" or "deploy"?  Please reword to be clear, and not assume that the installer VM was used.

mrsabath commented:
In fact, I suggest putting these changes on hold until @akanso finishes the updates to the examples. We are redesigning the tiny example.

MikeSpreitzer commented:
Yes, all of the changes here either intersect with the example revisions we just agreed or are mistaken reverts.

linsun commented:
Mike, thanks for the review.  I had accidentally reverted a few changes.  I also addressed your feedback on vagrant user only when installer VM used.

MikeSpreitzer commented on examples/tiny-example.md:
````diff
@@ -138,10 +138,17 @@ Use Ansible on the installer machine to begin the process of deploying
+vagrant ssh installer-tiny
+```
+
+If installer VM is used, execute the following ansible scripts as the vagrant user:
````
The playbook needs to be executed, regardless of whether the installer VM is being used.

MikeSpreitzer commented:
You missed my comments on new line 105 and new line 151, and I added a comment at new line 146.

linsun commented:
Thank you Mike, I *hopefully* have reworked your comment on 101, 151, and the new comment on running scripts for non-vagrant vm installer.  For the comment in 151, I hadn't tried it.  In the previously incorrect git clone cmd, I had to add the cluster_name to get it working.

MikeSpreitzer commented on examples/tiny-example.md:
````diff
@@ -138,14 +138,17 @@ Use Ansible on the installer machine to begin the process of deploying
       -e "envs=../examples/envs env_name=dev-vbox" )
 ```
 
-Use Ansible on the installer machine to deploy an OpenRadiant shard on
````
You are undoing the recent addition of this remark.  Please don't.

MikeSpreitzer commented on examples/tiny-example.md:
```diff
@@ -138,14 +138,17 @@ Use Ansible on the installer machine to begin the process of deploying
 an environment.  This will create the certificates and keys that are
 common throughout the environment, and deploy the API proxy.
 
+If you use the installer VM created from [Create Installer VM](../tiny-example.md#create-installer-vm) as the installer machine, use vagrant ssh to ssh into the installer VM and execute the Ansible scripts as the *vagrant* user:
```
What if "you" are _not_?
Why repeat https://github.com/containercafe/containercafe/blob/master/examples/tiny-example.md#fyi-on-ssh-to-vagrantvirtualbox-vms here?

MikeSpreitzer commented:
Please re-read the latest version of this doc.  You are still undoing recent work.

linsun commented:
Thank you Mike!  Hopefully I didn't revert anything this time!   I found a way to preview links to see if it is working so I can avoid the commits to try the links in the future. :)  Overall, I wish we could have less doc for standing up radiant in local env in the simplest scenario... I think we offer lots of choices which is great when people need choices/options, but it does took quite long to stand up a simple env.   It took me 1-1.5 days to stand up the working env for Kube.

MikeSpreitzer commented:
Yes, Lin, some of us discussed the example yesterday and decided to re-org into one example that is focused on showing what it is like for Maureen to use an OpenRadiant environment and another example that shows an operator (like the Conductors squad) what it is like to operate an OpenRadiant environment.  @akanso is working on the first of those two.

linsun commented:
Thank you Mike for merging / reviewing!   It would be great if we could make this simpler for people to try open radiant quickly, maybe even have a simple developer version.
